lowcar/devices/KoalaBear/PID.cpp

```diff
@@ -1,5 +1,7 @@
 #include "PID.h"
 
+#define MAX_TPS 2700.0   // maximum encoder ticks per second that a motor can go (determined empirically)
```

Comment:
In which cases would we need to change this? When we get new encoders? If this needs to be changed, how can someone measure the correct value?

Reply:
Yeah, when we get new encoders / new motors or potentially with the new PDB or power distribution system.. all I did was took my four motors, ran them at 1.0 duty cycle for 1 second 5 times each in both directions and recorded how far the encoders ticked, then took the average of all of those readings and it came out to like 2697 ish so I rounded it up to 2700 for simplicity.